src/api/spec/models/bs_request_spec.rb

```diff
@@ -565,4 +565,28 @@
       end
     end
   end
+
+  describe '#as_json' do
+    before do
+      submit_request.update(superseded_by: delete_request.id)
+    end
+
+    subject { submit_request.as_json }
+
+    it 'returns a json representation of a bs request' do
+      expect(subject['id']).to eq(submit_request.id)
```

Comment:
This would be a nice use case for the [include matcher](https://relishapp.com/rspec/rspec-expectations/v/3-7/docs/built-in-matchers/include-matcher)